lib/ansible/modules/cloud/spotinst/spotinst_aws_elastigroup.py

```diff
@@ -0,0 +1,1507 @@
+                         'adjustment_percentage',
+                         'batch_size_percentage',
+                         'cron_expression',
+                         'frequency',
+                         'grace_period',
+                         'task_type',
+                         'is_enabled',
+                         'scale_target_capacity',
+                         'scale_min_capacity',
+                         'scale_max_capacity')
+
+scaling_policy_fields = ('policy_name',
+                         'namespace',
+                         'metric_name',
+                         'dimensions',
+                         'statistic',
+                         'evaluation_periods',
+                         'period',
+                         'threshold',
+                         'cooldown',
+                         'unit',
+                         'operator')
+
+tracking_policy_fields = ('policy_name',
+                          'namespace',
+                          'source',
+                          'metric_name',
+                          'statistic',
+                          'unit',
+                          'cooldown',
+                          'target',
+                          'threshold')
+
+action_fields = ('action_type',
+                 'adjustment',
+                 'min_target_capacity',
+                 'max_target_capacity',
+                 'target',
+                 'minimum',
+                 'maximum')
+
+signal_fields = ('name',
+                 'timeout')
+
+multai_lb_fields = ('balancer_id',
+                    'project_id',
+                    'target_set_id',
+                    'az_awareness',
+                    'auto_weight')
+
+persistence_fields = ('should_persist_root_device',
+                      'should_persist_block_devices',
+                      'should_persist_private_ip')
+
+strategy_fields = ('risk',
+                   'utilize_reserved_instances',
+                   'fallback_to_od',
+                   'on_demand_count',
+                   'availability_vs_cost',
+                   'draining_timeout',
+                   'spin_up_time',
+                   'lifetime_period')
+
+ebs_fields = ('delete_on_termination',
+              'encrypted',
+              'iops',
+              'snapshot_id',
+              'volume_type',
+              'volume_size')
+
+bdm_fields = ('device_name',
+              'virtual_name',
+              'no_device')
+
+kubernetes_fields = ('api_server',
+                     'token')
+
+right_scale_fields = ('account_id',
+                      'refresh_token')
+
+rancher_fields = ('access_key',
+                  'secret_key',
+                  'master_host')
+
+chef_fields = ('chef_server',
+               'organization',
+               'user',
+               'pem_key',
+               'chef_version')
+
+az_fields = ('name',
+             'subnet_id',
+             'placement_group_name')
+
+opsworks_fields = ('layer_id')
```

Comment:
For these to be a tuple you need a trailing comma, like `opsworks_fields = ('layer_id',)` for all single-item tuples.

Reply:
Thanks, fixed.